### DISCUSSION TITLE
Translate language to Chinese,Traditional(zh-Hant)

### DISCUSSION THREAD

**Discussion by Henryliu880922** [Klocman/Bulk-Crap-Uninstaller#482] [General]
While Update the BCUninstaller, I found that you Release the language Pack, and still hasn't the Traditional Chinese Language, so I spend my time to translate it. I am Taiwanese,and I live in Taiwan,Thanks!
[Files to translate.zip](https://github.com/Klocman/Bulk-Crap-Uninstaller/files/11239841/Files.to.translate.zip)

**Comment by Wind-stormger** [comment]
It is best to submit a PR.

**Comment by Klocman** [comment]
Already merged in.
